Packages/KeyAppKit/Sources/KeyAppBusiness/Ethereum/EthereumAccountsService.swift

```diff
@@ -11,22 +11,27 @@ import KeyAppKitCore
 import SolanaPricesAPIs
 import Web3
 
-public final class EthereumAccountsService: NSObject, AccountsService, ObservableObject {
+public final class EthereumAccountsService: NSObject, AccountsService {
     public typealias Account = EthereumAccount
 
     private var subscriptions = [AnyCancellable]()
 
     private let accounts: AsyncValue<[Account]>
 
-    @Published public var state: AsyncValueState<[Account]> = .init(value: [])
+    private let _state: CurrentValueSubject<AsyncValueState<[Account]>, Never> = .init(.init(value: []))
```

Comment:
@sidorov-panda @qwerkly @lisemyon we need to discuss about naming of the Subject.
We have `stateSubject`, `_state`. Lets choose!
I think the _state might be confused with public state, but lets vote and choose!